docs/source/cuda/types.rst

```diff
@@ -0,0 +1,49 @@
+Types on CUDA target
+====================
+
+Note: This page is about types specific to CUDA targets. Most types on CPU targets are also
+available in CUDA target, for information on these types, see :ref:`numba-types`.
+
+Vector Types
+~~~~~~~~~~~~
+
+`CUDA Vector Types <https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#built-in-vector-types>`_
+are available in the kernel. There are 2 important distinctions from vector types in numba CUDA:
+
+First, the recommended names for vector types in Numba CUDA is formatted as ``base_typexN``,
+where `base_type` is the base type of the vector, and `N` is the number of elements in the vector.
```

Comment:
Sphinx formatting for code instead of emphasis:

```suggestion
where ``base_type`` is the base type of the vector, and ``N`` is the number of elements in the vector.
```